setup.sh

```diff
@@ -49,11 +49,11 @@ install(){
 # Run tasks to build the project for distribution.
 build(){
   echo 'Building project...'
-  grunt clean
-  grunt
+  gulp clean
```

Comment:
No you were right, I was saying we could `rm -rf` dist in the clean task above, but only so all the cleaning is organized into one function. It's a duplication of ability with the `gulp clean` task, but it's organized in a way that makes a bit more sense when looking at `setup.sh`. Right now we clean, then build, but within that build is another clean. 

IDK, maybe it's just me being overly organized :smile: